packages/dialtone-vue2/directives/tooltip_directive/tooltip.js

```diff
@@ -79,10 +79,11 @@ export const DtTooltipDirective = {
       },
     });
 
+    // eslint-disable-next-line complexity
     function setupTooltip (anchor, binding) {
       const tooltipId = anchor.getAttribute('data-dt-tooltip-id') || getUniqueString();
-      const message = binding.value;
-      const placement = binding.arg || DEFAULT_PLACEMENT;
+      const message = typeof binding.value === 'object' ? binding.value.text : binding.value;
```

Comment:
Checking if binding.value === 'object' can bring some errors as in JS `[]` is an object too. In that case `binding.value.text` will throw an error.

We'll need to make a better validation to avoid such cases.